### PR TITLE
Shut down clustercontroller clusters individually

### PR DESCRIPTION
**Pull request by jonmv**
@vekterli please review.

### CODE REVIEW

**Review by vekterli — DISMISSED** [review]
:+1: a couple of comments for consideration

File: clustercontroller-apps/src/main/java/com/yahoo/vespa/clustercontroller/apps/clustercontroller/ClusterController.java [hunk 1]
```diff
@@ -27,9 +28,10 @@ public class ClusterController extends AbstractComponent
 
     private static final Logger log = Logger.getLogger(ClusterController.class.getName());
     private final JDiscMetricWrapper metricWrapper;
+    private final Object monitor = new Object();
     private final Map<String, FleetController> controllers = new TreeMap<>();
-    private final Map<String, StatusHandler.ContainerStatusPageServer> status = new TreeMap<>();
-    private final AtomicInteger referents = new AtomicInteger();
+    private final Map<String, StatusHandler.ContainerStatusPageServer> status = new HashMap<>();
```
[the remark this answers is not in the document]
Comment (jonmv) [reply clustercontroller-apps/src/main/java/com/yahoo/vespa/clustercontroller/apps/clustercontroller/ClusterController.java:33]: Urgh, I looked for things like this, and missed this one :'( Thanks!

**Comment by jonmv** [comment]
PTAL.

**Review by vekterli — APPROVED** [review]
:+1:

**Pull request merged**